modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-storage-discovery_{context}"]
+= Automating discovery and provisioning for local storage devices
+
+The Local Storage Operator automates local storage discovery and provisioning. With this feature, you can simplify installation when dynamic provisioning is not otherwise available during deployment, such as with bare metal, VMware, or AWS store instances with attached devices.
+
+[IMPORTANT]
+====
+Automatic discovery and provisioning is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Use the following procedure to create discovery requests, automatically discover local devices, and automatically provision local volumes for selected devices.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+
+* Local disks are attached to {product-title} nodes.
+
+* You have access to the {product-title} web console and the `oc` command-line interface (CLI).
+
+.Procedure
+. To create a local volume discovery request from the web console:
+
+.. Navigate to *Operators* -> *OperatorHub* -> *Installed Operators*.
+
+.. Click on *Local Storage* from the Operators list.
+
+.. Select the *Local Volume Discovery Result* tab to display the list of results.
+
+.. Click *Create Local Volume Discovery Result*.
+
+.. Complete the required fields in *Form View* or click *YAML view* for full control.
+
+.. When finished, click *Create*. The new result is displayed.
+. To display a continuous list of available devices on a node:
+
+.. Log in to the {product-title} web console.
+
+.. Navigate to *Compute* -> *Nodes*.
+
+.. Click the node name that you want to open. The "Node Details" page is displayed.
+
+.. Select the *Disks* tab to display the list of the selected devices.
++
+The device list updates continuously as local disks are added or removed. You can filter the devices by name, status, type, model, capacity, and mode.
++
+[NOTE]
+====
+Results are deleted after they are removed from the node. Symlinks must be manually removed.
+====
+
+. To automatically provision local volumes for the discovered devices from the web console:
+
+.. Navigate to *Operators* -> *Installed Operators* and select *Local Storage* from the list of Operators.
+
+.. Select *Local Volume Set* -> *Create Local Volume Set*.
+
+.. Enter a volume set name and a storage class name.
+
+.. Choose *All nodes* or *Select nodes* to apply filters accordingly.
+
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1866748
For "all nodes", we need to clarify master nodes not included now.

Reply:
Added a note both here and below in the auto-disco step: 

> Only worker nodes are available, regardless of whether you filter using All nodes or Select nodes.